data/brands/amenity/pharmacy.json

```
@@ -3853,10 +3853,12 @@
       "id": "createsd-650eee",
       "locationSet": {"include": ["jp"]},
       "tags": {
+        "alt_name": "薬CREATE",
+        "alt_name:ja": "薬クリエイト",
         "amenity": "pharmacy",
-        "brand": "薬CREATE",
-        "brand:en": "CREATE",
-        "brand:ja": "薬クリエイト",
+        "brand": "クリエイトSD",
+        "brand:en": "CreateSD",
```

```suggestion
        "brand:en": "Create SD",
```
This has nothing to do with the issue I mentioned, but as far as the official website is concerned, it might be better to open a space in the English name.